CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This document follows the conventions laid out in [Keep a CHANGELOG][].
 ## [unreleased][]
 
 ### Added
+-   Optimized implicit assembly loading on module import, PythonEngine.ImplicitAssemblyLoading event added.
```

Comment:
@dmitriyse  please add reference to this pull request and issues from which you referred to this pull request.